### PR TITLE
CT-1322 Update trigger stats in-time calculations

### PR DESCRIPTION
**Pull request by stephenrichards**

### CODE REVIEW

File: app/models/case.rb [hunk 1]
```diff
@@ -444,6 +457,15 @@ def is_internal_review?
   end
 
   private
+  # determines whether or not the BU responded to flagged cases in time (NOT whether the case was responded to in time!)
+  # calculated as the time between the responding BY accepting the case and the disclosure team approving it.
+  def business_unit_responded_to_flagged_case_in_time?
+    responding_team_acceptance_date = transitions.where(event: 'accept_responder_assignment').last.created_at.to_date
```
Comment (misaka) [inline app/models/case.rb]: The ticket says:

> IF 10 days or less have passed from when the BU has been assigned to the case ...

What's implemented above appears to be using the date that the responder _accepted_ the assignment. Which on is correct?

Comment (stephenrichards) [reply app/models/case.rb]: Should be accepted: will change the comment.

Comment (misaka) [reply app/models/case.rb]: Just to follow-up, discussion IRL have shown that it is in fact on assignment, not acceptance. Code should be changed. :)

Comment (stephenrichards) [reply app/models/case.rb]: Nope - should be assigned now.  Code and tests changed

**Pull request merged**